<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDataframe">
    <property name="title">
     <string>Dataframe</string>
    </property>
    <widget class="QMenu" name="menuSave_current_tab">
     <property name="title">
      <string>Save current tab</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionto_pickle_tab"/>
     <addaction name="actionto_csv_tab"/>
     <addaction name="actionto_excel_tab"/>
    </widget>
    <widget class="QMenu" name="menuSave_all_tabs">
     <property name="title">
      <string>Save root</string>
     </property>
     <addaction name="actionto_pickle"/>
     <addaction name="actionto_csv"/>
     <addaction name="actionto_excel"/>
    </widget>
    <addaction name="actionDataframe_editor"/>
    <addaction name="separator"/>
    <addaction name="actionUpdate_all_tabs"/>
    <addaction name="actionUpdate_current_tab"/>
    <addaction name="separator"/>
    <addaction name="menuSave_current_tab"/>
    <addaction name="menuSave_all_tabs"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCurrent_tab_filter_history"/>
    <addaction name="separator"/>
    <addaction name="actionConsole"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionLink_viewers"/>
   </widget>
   <addaction name="menuDataframe"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockConsole">
   <property name="minimumSize">
    <size>
     <width>42</width>
     <height>200</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Console: Project Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionDataframe_editor">
   <property name="text">
    <string>Dataframe editor</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionCurrent_tab_filter_history">
   <property name="text">
    <string>Current tab filter history</string>
   </property>
  </action>
  <action name="actionUpdate_current_tab">
   <property name="text">
    <string>Update current tab</string>
   </property>
  </action>
  <action name="actionUpdate_all_tabs">
   <property name="text">
    <string>Update all tabs</string>
   </property>
  </action>
  <action name="actionto_pickle_tab">
   <property name="text">
    <string>to pickle</string>
   </property>
   <property name="toolTip">
    <string>Save current tab as a python pickle</string>
   </property>
  </action>
  <action name="actionto_csv_tab">
   <property name="text">
    <string>to csv</string>
   </property>
  </action>
  <action name="actionto_excel_tab">
   <property name="text">
    <string>to excel</string>
   </property>
  </action>
  <action name="actionto_pickle">
   <property name="text">
    <string>to pickle</string>
   </property>
  </action>
  <action name="actionto_excel">
   <property name="text">
    <string>to excel</string>
   </property>
  </action>
  <action name="actionto_csv">
   <property name="text">
    <string>to csv</string>
   </property>
  </action>
  <action name="actionLink_viewers">
   <property name="text">
    <string>Link viewers</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionConsole</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockConsole</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
